app/models/item.rb

```diff
@@ -5,22 +5,47 @@ class Item < ActiveRecord::Base
   validates :text, :list_id, presence: true
 
   scope :incompleted, -> { where.not('done') }
+  scope :completed, -> { where('done') }
+  scope :for_today, -> {
+    where('completed_at >= ? AND completed_at <= ?',
+    Date.today.to_time.beginning_of_day,
+    Date.today.to_time.end_of_day)
```

Comment:
@vlasiak, will [the following feature of rande condition](http://guides.rubyonrails.org/active_record_querying.html#range-conditions) work instead of texting sql here (see 2.3.2 item)?